### ISSUE TITLE
customMetadata not work in wdio v7 

### ISSUE THREAD

**Issue by Seunghyum** [wswebcreation/multiple-cucumber-html-reporter#150]
my pr is https://github.com/wswebcreation/multiple-cucumber-html-reporter/pull/151

**Environment (please complete the following information):**
 - multiple-cucumber-html-reporter: 1.18.0
 - Node.js version: 14.16.1
 - NPM version: 6.14.12
 - Platform name and version: Mac 
 - Cucucmber version: 7.1.0

**Config of multiple-cucumber-html-reporter**
```
onComplete(exitCode, config, capabilities, results) {
  report.generate({
    jsonDir: `${BASE_DIR}/json`,
    reportPath: `${BASE_DIR}/html`,
    openReportInBrowser: true,
    displayReportTime: true,
    displayDuration: true,
    pageFooter: "Zigbang E2E Test",
    customMetadata: true,
    metadata: [
      {name: 'OS', value: `${platformName} ${platformVersion}` },
      {name: 'Device', value: deviceName },
    ]
  })
}
```
**Describe the bug**
customMetadata not work in wdio v7 
my pr is 

**To Reproduce**
set customMetadata: true
and set metadata like
```
metadata: [
  {name: 'OS', value: `${platformName} ${platformVersion}` },
  {name: 'Device', value: deviceName },
]
```

**Expected behavior**
it works in wdio v7

**Comment by BaCaRoZzo** [comment]
@wswebcreation wasn't this issue closed by #166 but automation didn't kick in because the usual GH syntax wasn't used on the PR?
I think we can close this. :thinking:

**Comment by wswebcreation** [comment]
Correct, thanks!
